tests/boot/grub_test_snapshot.pm

```diff
@@ -14,6 +14,11 @@ use testapi;
 
 sub run() {
     my $self = shift;
+
+    assert_screen "inst-bootmenu", 30;
+    sleep 2;
+    send_key "ret";    # boot
+    sleep 10; # wait some seconds for booting.
```

Comment:
you're doing a 30 second wait as part of the assert screen for grub2..why wait for 10 seconds before doing it?